openpilot/dummy_pip_package.py

```diff
@@ -0,0 +1,3 @@
+
+
```

Comment:
@haraschax I'd rather just start doing this properly instead of making a dummy package. is #29338 enough to make this work?